### PR TITLE
Make 'for' configurable for falco exporter prometheus rules

### PR DESCRIPTION
**Pull request by TheChef23**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area falco-exporter-chart

**What this PR does / why we need it**:

This PR makes the `for` field configurable within the helm values. 

**Which issue(s) this PR fixes**:

N/a

### CODE REVIEW

**Comment by Issif** [comment]
Hi,

to have a valid PR, you have to update the changelog, bump up the chart version and run `make docs-falco-exporter`.

> [!WARNING]
The falco-exporter will be deprecated in the next months, Falco now exposes a metrics endpoint

**Comment by TheChef23** [comment]
> Hi,
> 
> to have a valid PR, you have to update the changelog, bump up the chart version and run `make docs-falco-exporter`.
> 
> Warning
> 
> The falco-exporter will be deprecated in the next months, Falco now exposes a metrics endpoint

Hi,

I've run the make command and bumped the version.

**Review by Issif — COMMENTED** [review]
Can you also update the changelog please.

**Comment by TheChef23** [comment]
Excusez moi, fixed.

File: charts/falco-exporter/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ numbering uses [semantic versioning](http://semver.org).
 
 ## v0.11.0
```
Comment (Issif) [inline charts/falco-exporter/CHANGELOG.md:6]:
```suggestion
## v0.12.0
```

**Review by Issif — COMMENTED** [review]
Last point, could you rebase on master and squash your commits please

**Comment by TheChef23** [comment]
> Last point, could you rebase on master and squash your commits please

Done, thanks for your patience!

**Review by Issif — APPROVED** [review]

**Pull request merged**